src/Admin/ProductBlocksService.php

```diff
@@ -91,34 +93,55 @@ function () {
 		$template_area = 'product-form';
 
-		// https://github.com/woocommerce/woocommerce/blob/8.3.0/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php#L361
-		$block_id = 'product-attributes-section';
+		// https://github.com/woocommerce/woocommerce/blob/8.3.0/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php#L18
+		// https://github.com/woocommerce/woocommerce/blob/8.3.0/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php#L18
+		$block_id = 'general';
 
 		add_action(
 			"woocommerce_block_template_area_{$template_area}_after_add_block_{$block_id}",
-			function ( BlockInterface $attributes_section ) {
-				// Please note that the simple and variable product types use the same product block template 'simple-product'.
-				if ( 'simple-product' !== $attributes_section->get_root_template()->get_id() ) {
-					return;
-				}
-
-				$section = $this->add_section( $attributes_section );
-				$this->add_blocks( $section );
-			}
-		);
-
-		// https://github.com/woocommerce/woocommerce/blob/8.3.0/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php#L161
-		$block_id = 'product-variation-images-section';
+			function ( BlockInterface $general_group ) {
+				/** @var Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates\ProductFormTemplateInterface */
+				$template = $general_group->get_root_template();
 
-		add_action(
-			"woocommerce_block_template_area_{$template_area}_after_add_block_{$block_id}",
-			function ( BlockInterface $images_section ) {
-				if ( ! $this->is_variation_template( $images_section ) ) {
+				// Please note that the simple and variable product types use the same product block template 'simple-product'.
+				if ( 'simple-product' !== $template->get_id() && ! $this->is_variation_template( $general_group ) ) {
 					return;
 				}
 
-				$section = $this->add_section( $images_section );
-				$this->add_blocks( $section );
+				/** @var Automattic\WooCommerce\Internal\Admin\Features\ProductBlockEditor\ProductTemplates\Group */
+				$group = $template->add_group(
+					[
+						'id'         => 'google-listings-and-ads-group',
+						'order'      => 100,
+						'attributes' => [
+							'title' => __( 'Google Listings & Ads', 'google-listings-and-ads' ),
+						],
+					]
+				);
+
+				/** @var Automattic\WooCommerce\Internal\Admin\Features\ProductBlockEditor\ProductTemplates\Section */
+				$channel_visibility_section = $group->add_section(
+					[
+						'id'         => 'google-listings-and-ads-channel-visibility-section',
+						'order'      => 1,
+						'attributes' => [
+							'title' => __( 'Channel visibility', 'google-listings-and-ads' ),
+						],
+					]
+				);
+
+				/** @var Automattic\WooCommerce\Internal\Admin\Features\ProductBlockEditor\ProductTemplates\Section */
```

Comment:
I think it should be `SectionInterface`

Reply:
Changed in https://github.com/woocommerce/google-listings-and-ads/pull/2179/commits/666cc9dcef530943e109981f5647af95e9a1a942.